ballerina/llm_client_azure_open_ai.bal

```diff
@@ -69,10 +91,18 @@ isolated distinct client class AzureOpenAIModel {
             return error("No completion choices");
         }
 
-        string? resp = choices[0].message?.content;
-        if resp is () {
+        ChatCompletionMessageToolCalls? toolCalls = choices[0].message?.tool_calls;
+        
+        if toolCalls is () {
             return error("No completion message");
         }
-        return parseResponseAsType(resp, expectedResponseTypedesc);
+
+        string? resp = toolCalls[0].'function.arguments;
+
+        if resp is () || resp == RESPONSE_WITH_EMPTY_PARAMETERS  {
+            return error("No completion message");
```

Comment:
Could possibly be updated now.

Reply:
Do you mean https://github.com/ballerina-platform/module-ballerina-np/pull/16#discussion_r2062387531?

Reply:
No, it isn't simply a completion anymore. But we shouldn't expose tool calling either. So can be a generic error now.